include/libdnf/advisory/advisory_module.hpp

```diff
@@ -31,8 +31,8 @@ class AdvisoryModule {
     AdvisoryModule(const AdvisoryModule & src);
     AdvisoryModule & operator=(const AdvisoryModule & src);
 
```

Comment:
Is this commit still worth it with the new `ImplPtr` class? We want to move over to using that instead...